frontend/src/settings/display/ExampleDisplay.tsx

```diff
@@ -0,0 +1,35 @@
+import { Box, Button } from '@mui/material'
+import { ThemeProvider } from '@mui/material/styles'
+import { patchCurrentUserSettings, useGetCurrentUserSettings } from 'actions/user'
+import { ThemeKey } from 'src/theme'
+
+interface ExampleDisplayProps {
+  theme: ThemeKey
+}
+export default function ExampleDisplay({ theme }: ExampleDisplayProps) {
+  const { mutateUserSettings } = useGetCurrentUserSettings()
+
+  async function setTheme() {
```

Comment:
I'd prefer to keep the `setSomething` naming convention for updating react state vars. Consider renaming this function to something like `handleUpdateTheme`